client/components/infinite-list/scroll-helper.js

```diff
@@ -477,6 +466,6 @@ assign ( ScrollHelper.prototype, {
 			} );
 		}.bind( this ), 0 );
```

Comment:
The setTimeout callback here could also be replaced with an `=>` func which should let us remove the `.bind(this)`